### ISSUE TITLE
Next/future/image blur placeholder have undefined to view box SVG width & height

### ISSUE THREAD

**Issue by Balkeo** [vercel/next.js#41393]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

    Operating System:
      Platform: linux
      Arch: x64
      Version: #25-Ubuntu SMP Wed Mar 30 15:54:22 UTC 2022
    Binaries:
      Node: 18.9.0
      npm: 8.19.1
      Yarn: 1.22.10
      pnpm: N/A
    Relevant packages:
      next: 12.3.1
      eslint-config-next: 12.3.1
      react: 18.2.0
      react-dom: 18.2.0

### What browser are you using? (if relevant)

Brave Version 1.44.108 Chromium: 106.0.5249.103

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

Wrong SVG viewBox generated when using placeHolder='blur' with image filling his parent

- ### The blur base64
    - ![](https://raw.githubusercontent.com/Balkeo/next-future-image-issue/main/src/blur.svg)
- ### The SVG using the blur base64
    - ![](https://raw.githubusercontent.com/Balkeo/next-future-image-issue/main/src/render-by-browser.png)

As we can see in the `src/blur.svg` the viewBox have weird values

```SVG
<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 undefined undefined'>
    <filter id='b' color-interpolation-filters='sRGB'>
        <feGaussianBlur stdDeviation='20'/>
        <feComponentTransfer>
            <feFuncA type='discrete' tableValues='1 1'/>
        </feComponentTransfer>
    </filter>
    <image filter='url(#b)' x='0' y='0' height='100%' width='100%' href='data:image/jpeg;base64[...]'/>
</svg>
```

So the svg have this weird artifacts and the black borders

### Expected Behavior

I expected the placeHolder to be rendered without artefacts and with a corect viewBox for the SVG not undefined in width & height

### Link to reproduction

https://next-future-image-issue-sigma.vercel.app/

### To Reproduce

A repo with all the need to reproduce 
https://github.com/Balkeo/next-future-image-issue

**Comment by balazsorban44** [comment]
Looks like a duplicate of #40419

**Comment by shouze** [comment]
@balazsorban44 maybe, not sure... but probably related :wink: 
- In #40419 the generated `width` property of the SVG was invalid (100%25)
- In this issue this is the viewbox with the 2 `undefined` properties

**Comment by ElektrikSpark** [comment]
![next-future-image-blur-bug](https://user-images.githubusercontent.com/24904780/195974444-fad1f2bd-0be5-4b9a-9954-f55adfd43e5c.png)
Similar issue. Here is a look at images from Sanity loading in. Seems that the aspect ratio of the image can make it worse than another image?

**Comment by styfle** [comment]
Thanks for reporting! We'll get this fixed in #41573
